tests/src/test/scala/com/nvidia/spark/rapids/timezone/TimeZoneSuite.scala

```diff
@@ -161,18 +181,24 @@ class TimeZoneSuite extends SparkQueryCompareTestSuite {
           .set("spark.sql.datetime.java8API.enabled", "true"))
 
     // get result from TimeZoneDB
-    val actualRet = withResource(createColumnVector(epochSeconds)) { inputCv =>
-      TimeZoneDB.fromUtcTimestampToTimestamp(
-        inputCv,
-        ZoneId.of(zoneStr))
+    withResource(createColumnVector(epochSeconds)) { inputCv =>
+      val actualRet = if (useGPU) {
```

Comment:
No need to compare CPU ```TimezoneDB``` result against Spark.

Reply:
this includes part of another PR #9739 which should be merged first after https://github.com/NVIDIA/spark-rapids-jni/pull/1553 is merged